pynestml/codegeneration/nest_code_generator.py

```diff
@@ -170,7 +171,16 @@ def __init__(self, options: Optional[Mapping[str, Any]] = None):
         self.setup_printers()
 
     def run_nest_target_specific_cocos(self, neurons: Sequence[ASTModel], synapses: Sequence[ASTModel]):
+        for neuron in neurons:
```

Comment:
I think this should work too:
```suggestion
        for model in neurons + synapses:
```